### PR TITLE
Check and do something about all "TODO" comments in code

### PR DESCRIPTION
**Pull request by luc10921**
**Summary or solution description**
Removed some TODO messages and added the task codes to the ones that remain

### CODE REVIEW

**Comment by lllwvlvwlll** [comment]
Task linked: [CU-864ec9c4g Check and do something about all "TODO" comments in code](https://app.clickup.com/t/864ec9c4g)

File: boa3_test/test_drive/model/network/payloads/signer.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def __init__(self, account: UInt160, scopes: WitnessScope = WitnessScope.CalledB
         self._scopes: WitnessScope = scopes
         self._allowed_contracts: List[UInt160] = []
         self._allowed_groups: List[bytes] = []
-        # TODO: implement allowed witness rules
+        # TODO: implement allowed witness rules #86a1cu3y6
```
Comment (meevee98) [inline boa3_test/test_drive/model/network/payloads/signer.py:18]: We won't work on this, you can remove this TODO

File: boa3_test/test_drive/model/network/payloads/signer.py [hunk 2]
```diff
@@ -45,7 +45,7 @@ def from_json(cls, json: Dict[str, Any]) -> Signer:
                                      if 'allowedcontracts' in json else [])
         signer._allowed_groups = ([from_hex_str(group_hex) for group_hex in json['allowedgroups']]
                                   if 'allowedgroups' in json else [])
-        # TODO: implement allowed witness rules
+        # TODO: implement allowed witness rules #86a1cu3y6
```
Comment (meevee98) [inline boa3_test/test_drive/model/network/payloads/signer.py:48]: We won't work on this, you can remove this TODO

File: boa3/internal/analyser/importanalyser.py [hunk 3]
```diff
@@ -153,7 +153,7 @@ def _find_package(self, module_origin: str, origin_file: Optional[str] = None):
                     self.recursive_import = True
                     return
 
-                # TODO: only user modules and typing lib imports are implemented
+                # TODO: only user modules and typing lib imports are implemented #86a1ct917
```
Comment (meevee98) [inline boa3/internal/analyser/importanalyser.py:156]: We won't work on this, you can remove this TODO

**Review by meevee98 — CHANGES_REQUESTED** [review]

**Review by meevee98 — APPROVED** [review]

**Pull request merged**